core/src/test/scala/integration/kafka/api/SslAdminIntegrationTest.scala

```diff
@@ -41,7 +41,7 @@ import org.junit.{Assert, Test}
 import scala.collection.JavaConverters._
 import scala.collection.mutable
 
-object SslAdminClientIntegrationTest {
+object SslAdminIntegrationTest {
```

Comment:
Is this name clearer? It can be confused with a broker test.

Reply:
I made the change for consistency with the `AdminClient` to `Admin` change. It seems clear enough either way.